client/i18n/locales/english/intro.json

```
@@ -315,126 +316,129 @@
         "title": "Build a Cash Register Project",
         "intro": [
           "This is one of the required projects to earn your certification.",
-          "For this project, you will build a cash register web app."
+          "For this project, you'll build a cash register web app."
         ]
       },
       "build-a-palindrome-checker-project": {
         "title": "Build a Palindrome Checker Project",
         "intro": [
           "This is one of the required projects to earn your certification.",
-          "For this project, you will build an application that checks if a word is a palindrome."
+          "For this project, you'll build an application that checks whether a given word is a palindrome."
         ]
       },
       "build-a-roman-numeral-converter-project": {
         "title": "Build a Roman Numeral Converter Project",
         "intro": [
           "This is one of the required projects to claim your certification.",
-          "For this project, you will build an application that converts integers to Roman numerals."
+          "For this project, you'll build an application that converts integers to Roman numerals."
         ]
       },
       "build-a-telephone-number-validator-project": {
         "title": "Build a Telephone Number Validator Project",
         "intro": [
           "This is one of the required projects to claim your certification.",
-          "For this project, you will build an application that checks if a number is a valid United States phone number."
+          "For this project, you'll build an application that checks if a number is a valid United States phone number."
         ]
       },
       "learn-basic-javascript-by-building-a-role-playing-game": {
         "title": "Learn Basic JavaScript by Building a Role Playing Game",
         "intro": [
-          "JavaScript is a scripting language you can use to make web pages interactive. It is one of the core technologies of the web, along with HTML and CSS, and is supported by all modern browsers.",
-          "In this practice project, you'll learn fundamental programming concepts in JavaScript by building a role playing game. You'll learn to work with arrays, strings, objects, functions, loops, if/else statements, and more."
+          "JavaScript is a powerful scripting language that you can use to make web pages interactive. It's one of the core technologies of the web, along with HTML and CSS. All modern browsers support JavaScript.",
+          "In this practice project, you'll learn fundamental programming concepts in JavaScript by coding your own Role Playing Game. You'll learn how to work with arrays, strings, objects, functions, loops, if/else statements, and more."
```

```suggestion
          "In this practice project, you'll learn fundamental programming concepts in JavaScript by coding your own Role Playing Game. You'll learn how to work with arrays, strings, objects, functions, loops, <code>if/else</code> statements, and more."
```

These are technically tokens so should not be translated.